### PR TITLE
Update README.md with traditional EPO example, caveats, and `@task.ve…

### PR DESCRIPTION
**Pull request by RNHTTR**
…nv` documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,7 +211,13 @@ PYENV 3.10 venv1
 
 ### `@task.venv` decorator
 
-TODO! Write the decorator, then fill out docs!
+The `@task.venv` decorator wraps the [ExternalPythonOperator](https://airflow.apache.org/docs/apache-airflow/stable/howto/operator/python.html#externalpythonoperator). The decorator does a few things:
+1. It assigns the decorated function as the 'ExternalPythonOperator`'s callable
```
Comment (kaxil) [inline README.md:215]:
```suggestion
1. It assigns the decorated function as the '`ExternalPythonOperator`'s callable
```

File: README.md [hunk 2]
```diff
@@ -19,6 +19,32 @@ This repo provides a nice packaged solution to it, that plays nicely with Docker
 
 ## Synopsis
 
+### Caveats
+#### Novel Python Syntax
```
Comment (ashb) [inline README.md:23]: This shouldn't be the first thing in the README -- people don't know what this repo is yet.

Comment (ashb) [reply README.md:23]: Maybe move both of these to inside the `### Requirements` section of the doc?

File: README.md [hunk 3]
```diff
@@ -144,7 +211,13 @@ PYENV 3.10 venv1
 
 ### `@task.venv` decorator
 
-TODO! Write the decorator, then fill out docs!
+The `@task.venv` decorator wraps the [ExternalPythonOperator](https://airflow.apache.org/docs/apache-airflow/stable/howto/operator/python.html#externalpythonoperator). The decorator does a few things:
+1. It assigns the decorated function as the `ExternalPythonOperator`'s callable
+2. It assigns the string passed to the decorator to the `ExternalPythonOperator`'s `python` parameter.
```
Comment (ashb) [inline README.md:216]: That's not quite what it does -- it actually looks up full path form the environment variables that the buildkit puts in to the image.

Maybe we just say something to the tune of "it looks up the absolute path of the virtual env ..."

**Review by kaxil — APPROVED** [review]

**Pull request merged**